TempleEight/docs/guide/business-logic.md

```diff
@@ -76,21 +79,225 @@ import (
 )
 
 func (env *env) setup(router *mux.Router) {
-	env.hook.BeforeCreate(beforeCreateHook)
+	env.hook.BeforeCreate(ourCustomBeforeHook)
 }
 
-func beforeCreateHook(env *env, req createExampleServiceRequest, input *dao.CreateExampleServiceInput) *HookError {
+func ourCustomBeforeHook(env *env, req createExampleServiceRequest, input *dao.CreateExampleServiceInput) *HookError {
 	return nil
 }
 ```
 
-// TODO
 ## Modifying the DAO request
+Now that we have defined our custom hook, we can start populating it with additional logic.
+To start, we will show how you can modify the DAO request.
```

Comment:
Have we defined `DAO` anywhere?

Reply:
Yeah wasn't sure what to do about that, was thinking the Go section would cover it - @smithwjv? I can add a reference here

Reply:
Yeah Go section should have it